### DISCUSSION TITLE
Unable to resolve dependency [Parameter #0 [ <required> $value ]] in class App\Livewire\SelectOptionParticular

### DISCUSSION THREAD

**Discussion by KenlymarP** [livewire/livewire#7856] [1. Help]
Hello! I am developing a school project on Letters Rogatory for a Ministry in my country where I need to show three inputs depending on the user's choice of the "particular" option. I am new to using Livewire and Laravel.

My Livewire component looks like this: 

![image](https://github.com/livewire/livewire/assets/137658742/5dab47bf-6a9f-4773-8ef4-7341d2ffdcfe)

and my blade like this:

![image](https://github.com/livewire/livewire/assets/137658742/771dea36-59ab-4efd-8f98-9a4a28b08051)

and this is the error I get: 
![image](https://github.com/livewire/livewire/assets/137658742/ed4f1da8-68a7-4e06-82f3-4461f47f929b)

**Comment by joshhanley** [comment]
@KenlymarP Please don't paste screenshots of code (screenshots of error messages is fine) as it makes it hard for us to edit your code to help you. 

Please paste code in code blocks. 
When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

**Comment by joshhanley** [comment]
@KenlymarP the problem is you have a `$value` parameter on your `OpcionSeleccionada($value)` method. But you are calling it as `wire:change="OpcionSeleccionada"` without passing a value in. Try changing it to `wire:change="OpcionSeleccionada($event.target.value)"`.
